### DISCUSSION TITLE
Deployig not working correctly

### DISCUSSION THREAD

**Discussion by rodriguez-p** [alshedivat/al-folio#2436] [Q&A] [answered]
Hi, I have been following the install guide using the recommended approach. Everything works perfectly when running the site locally (using 'bundle exec jekyll serve') but when I commit and push the changes it does not show any errors in the deployment but the site does not look like the local version. I can share the repo if needed.

Also, the link it shows is https://my-repo.github.io/my-repo. Is it possible to only have the first part?

Thanks!

**Comment by george-gca** [comment, marked as answer]
Duplicate of #2074
